src/components/autofill/preview-item/preview-item.component.js

```diff
@@ -42,12 +56,14 @@ const PreviewItem = ({
         </div>
         <div>
           <div className="flex-row font-16">
-            <Highlighter
-              highlightClassName="highlighted"
-              searchWords={searchWords}
-              autoEscape={true}
-              textToHighlight={name}
-            />
+            <Link to={searchResultsLink}>
```

Comment:
I like how Link is a more declarative solution,
but doesn't it add some default styling to the item, like underline and active/visited colors?

Reply:
Yes, it does add some default styles.

Reply:
To keep the component styles wrapped by `Link`, 
we can maybe try something like this:
https://github.com/WildAid/o-fish-web/pull/327#issuecomment-713894745